### ISSUE TITLE
Alert me when my favorite piece becomes available on the secondary market

### ISSUE THREAD

**Issue by technosmith** [hicetnunc2000/hicetnunc#789]
**Is your feature request related to a problem? Please describe.**
When collectors are looking for a piece/s that is no longer available “not for sale” they would need to go into the specific object every time to see the availability of that object. Collectors are looking to acquire many objects that are not for sale, it is difficult to manage.

**Describe the solution you'd like**
It'll be good to see be able to click on a “wish list” icon  displayed when an item is “not for sale” anymore so that that “wish list” item can be associated with the wallet. This item could then be visible when connecting to the HEN site with that wallet.

**Describe alternatives you've considered**
This could probably be achieved with a telegram bot but it would be better to  managed within the HEN ecosystem.

**Additional context**
![Untitled](https://user-images.githubusercontent.com/18900335/120102334-c6ae8980-c14a-11eb-839c-1dae64539bfe.jpg)

**Comment by kylegrover** [comment]
One question/thought I've had about this - should an objkts 'wish list' status persist after you've been notified of a listing? For example you wish list an item, get an email (or whatever) that it's available, and by the time you check it's already sold out again. For convenience I want to say it stays until you buy it, but for the servers sake that might get crazy. 

I also think functionality like this should cost some small amount of hdao or tez to use.

**Comment by technosmith** [comment]
Makes sense that there be some Tez/hDAO commitment for a feature like this. It will probably reduce its use to the more committed collectors in turn reduce the tech resource needs for the feature. I would prefer persist until buy.

**Comment by sf-pixel** [comment]
Why not just skip server completely? Just save the wishlist in local storage so it persists through sessions, then it can re-fetch the swap status every X minutes and push a desktop notif when swapped.

Could use service workers to accomplish this but I'm no expert with their limitations.

Feel like anything other than bare-bones wishlisting goes against what people like about hen.
